src/views/layouts/main.hbs

```diff
@@ -66,6 +64,9 @@
                             <li class="nav-item">
                                 <a class="nav-link ms-5" href="/player">Spieler</a>
                             </li>
+                            <li class="nav-item">
+                                <a class="nav-link ms-5" href="/certificate">Urkunden</a>
```

Comment:
Dont know if we need it here - can be placed in Setting in my opinion keeping the header not to full of items

Reply:
Ok i have made a Card for Klinger but this is a quick change later where the certificate page point is